python.rst

```diff
@@ -2,36 +2,93 @@
  Python Style Guide
 ====================
 
-Goal
+Background
 ====
 
-The goal of this style-guide is to standardize the motif of Python at
-Oscar in a way that reduces the risk of bugs while increasing our
-ability to utilize the richer features of Python when necessary. In
-addition, it reduces the risk exposure from the many third party
-libraries we use. Finally, if it is true that reading code is 90% of
-programming, then we should favor readability over writability.
+The goal of this document is to guide the style of Python code at
+Oscar to reduce bugs and improve readability.
 
-Example: `Import packages and modules only, do not import objects or
-functions directly`_ may seem constrictive or verbose, but it allows
-objects in module namespaces to be mutated during execution without
-unexpected behavior due to object duplication. Examples of this in our
-current code-base are sqlalchemy-continuum and tagging.
+Many teams use the `black <https://github.com/psf/black>`_
+auto-formatter to avoid having to worry (or even worse, argue!) about
+formatting. We recommend using it, but it does not cover everything
+here.
 
 Code of Conduct
 ===============
 
-- Be polite and respectful.
-- Don’t make assumptions—be clear and concise.
-- Your co workers deserve a response to comments, questions or
```

Comment:
can we leave this line in?

Reply:
Thoughts on splitting it out to a separate doc and linking it here? I agree we should recommend conduct, it just seems like a different kind of thing than python style or the Boy Scout rule.

Reply:
anyway: put it back in for now